tools/bin/ariaDownload.py

```diff
@@ -188,6 +196,74 @@ def _fmt_dst(self):
+                         'client': 'BO_n7nTIlMljdvU6kRRB3g',
+                         'redir': 'https://auth.asf.alaska.edu/login'}
+       if self.inps.user is not None and self.inps.passw is not None:
+           user  = self.inps.user
+           passw = self.inps.passw
+       elif op.exists(op.join(op.expanduser('~'), '.netrc')):
+           # log.info('Obtaining user/pass from .netrc')
+           user, _,  passw = netrc.netrc().authenticators('urs.earthdata.nasa.gov')
+       else:
+           # resort to ASF credential checks (will prompt for input if can't find the cookiejar)
+           return None
+
+       # Build URS4 Cookie request
+       auth_cookie_url = f"{asf_urs4['url']}?client_id={asf_urs4['client']}&redirect_uri={asf_urs4['redir']}&response_type=code&state="
+       user_pass = base64.b64encode(bytes(f'{user}:{passw}', 'utf-8')).decode('utf-8')
+
+       # Authenticate against URS, grab all the cookies
+       cookie_jar = MozillaCookieJar()
+       opener     = build_opener(HTTPCookieProcessor(cookie_jar), HTTPHandler(), HTTPSHandler(**{}))
+       request    = Request(auth_cookie_url, headers={'Authorization': f'Basic {user_pass}'})
+
+       # Watch out cookie rejection!
+       try:
+          response = opener.open(request)
+       except HTTPError as e:
+          if "WWW-Authenticate" in e.headers and "Please enter your Earthdata Login credentials" in e.headers["WWW-Authenticate"]:
+             print (" > Username and Password combo was not successful. Please try again.")
+             return False
+          else:
+             # If an error happens here, the user most likely has not confirmed EULA.
+             print ('\nThere was an error obtaining a download cookie')
+             print ('Most likely you lack permission to download data from the ASF Datapool.')
+             print ('\n\nNew users: you must first log into Vertex and accept the EULA. In addition, your Study Area must be set at Earthdata https://urs.earthdata.nasa.gov')
+             os.sys.exit(-1)
+
+       except URLError:
+          print ('\nThere was a problem communicating with URS, unable to obtain cookie')
+          print ('Try cookie generation later.')
+          os.sys.exit(-1)
+
+       # Did we get a cookie?
+       if check_cookie_is_logged_in(cookie_jar):
+          #COOKIE SUCCESS!
+          cookie_jar.save(cookie_jar_path)
+          return True
+
+       # if we aren't successful generating the cookie, nothing will work. Stop here!
+       print ('WARNING: Could not generate new cookie! Cannot proceed. Please check credentials and/or .netrc.')
+       print (f'Response was {response.getcode()}')
+       print ('\n\nNew users: you must first log into Vertex and accept the EULA. In addition, your Study Area must be set at Earthdata https://urs.earthdata.nasa.gov')
+       os.sys.exit(-1)
+
+def check_cookie_is_logged_in(cj):
+    """Make sure successfully logged into URS; try to get cookie"""
```

Comment:
Codacy found an issue: [First line should end with a period, question mark, or exclamation point (not 'e') (D415)](https://app.codacy.com/gh/aria-tools/ARIA-tools/pullRequest?prid=5918971)